doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -539,6 +539,112 @@ Requesting information
    fixpoint and cofixpoint is violated at some time of the construction
    of the proof without having to wait the completion of the proof.
 
+.. _showing_diffs:
+
+Showing differences between proof steps
+---------------------------------------
+
+
+Coq can automatically highlight the differences between successive proof steps.
+For example, the following screen shots of coqtop and CoqIDE show the same
+“intros” tactic.  This tactic creates two new hypotheses, highlighted in green.
+The conclusion is entirely in pale green because it’s changed, but no tokens were added
+to it.  The third screen shot uses the "removed" option, so it shows the conclusion a
+second time with the old text, with deletions marked in red.  Also, since the hypotheses are
+new, no line of old text is shown for them.
+
+.. _diffs_coqtop_on:
+
+  .. image:: ../_static/diffs-coqtop-on.png
+     :alt: coqtop with Set Diffs on
+
+.. _diffs_coqide_on:
+
+  .. image:: ../_static/diffs-coqide-on.png
+     :alt: |CoqIDE| with Set Diffs on
+
+.. _diffs_coqide_removed:
+
+  .. image:: ../_static/diffs-coqide-removed.png
+     :alt: |CoqIDE| with Set Diffs removed
+
+**How to enable diffs**
+
+.. opt:: Set Diffs %( "on" %| "off" %| "removed" %)
+.. This ref doesn't work: :opt:`Set Diffs %( "on" %| "off" %| "removed" %)`
+
+The “on” option highlights added tokens in green, while the “removed” option
+additionally reprints items with removed tokens in red.  Unchanged tokens in
+modified items are shown with pale green or red.  (Colors are user-configurable.)
+
+For coqtop, showing diffs can be enabled when starting coqtop with the
+``-diffs on|off|removed`` command-line option or with the ``Set Diffs``
+command within Coq.  You will need to provide the ``-color on|auto`` command-line option when
+you start coqtop in either case.
+
+Colors for coqtop can be configured by setting the ``COQ_COLORS`` environment
```

Comment:
This is mostly repetitive with the existing documentation of this environment variable. Can't you rather add a reference to it and improve / expand this documentation (use `.. seealso::`).

Reply:
OK, but I think it would be a `:see`, right?

Reply:
I don't know what is a `:see`. But you can you (top-level) `.. seealso::` directive (if you want): it works like `.. note::`.